src/main/java/com/serhiiklymchuk/gmailpet/config/AsyncConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.serhiiklymchuk.gmailpet.config;
+
+import com.serhiiklymchuk.gmailpet.domain.User;
+import com.serhiiklymchuk.gmailpet.dto.MessageFormDto;
+import com.serhiiklymchuk.gmailpet.exception.SendMessageException;
+import com.serhiiklymchuk.gmailpet.service.MessageService;
+import com.serhiiklymchuk.gmailpet.service.UserService;
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.AsyncConfigurer;
+
+@Configuration
+public class AsyncConfig implements AsyncConfigurer {
+
+    private static final Long GMAIL_SUPPORT_ID = 1L;
+
+    private final UserService userService;
+
+    private final ApplicationContext context;
+
+    public AsyncConfig(UserService userService, ApplicationContext context) {
+
+        this.userService = userService;
+        this.context = context;
```

Comment:
Get bean from the context here and assign it to to the field of `MessageService` type